cmd/vm_start.go

```diff
@@ -18,9 +18,18 @@ var vmStartCmd = &cobra.Command{
 			return cmd.Usage()
 		}
 
+		var rescueProfileP *string
```

Comment:
I think you don't have to define a pointer for this variable. You can pass `rescueProfile` directly to `startVirtualMachine` (like the `vmName` parameter). 
Because of the `omitempty` flag in the `StartVirtualMachine` call (here https://github.com/exoscale/egoscale/pull/401/files#diff-9d7e1c41fa657c6ea52f380e2d0c794dR318) if the string is empty the field will be discarded.

Reply:
Thanks, I wasn't sure about the behavior